js/src/files-mfs.js

```diff
@@ -37,10 +37,10 @@ module.exports = (common) => {
     after((done) => common.teardown(done))
 
     describe('.mkdir', function () {
-      it('make directory on root', (done) => {
+      it('make directory on root', function(done) {
```

Comment:
This fails on linting.

Reply:
?
The lint pass locally and on travis.
Do you want `function (done)` with a space instead ?

Reply:
@MichaelMure weird, I could bet that standard wouldn't accept that. Yeah I do want the whitespace to make it consistent :)